testing/tests/keyboard.lua

```diff
@@ -0,0 +1,73 @@
+-- love.keyboard
+-- @NOTE we can't test this module fully as it's hardware dependent
+-- however we can test methods do what is expected and can handle certain params
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+------------------------------------METHODS-------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+
+
+-- love.keyboard.getKeyFromScancode
+love.test.keyboard.getKeyFromScancode = function(test)
+  local key = love.keyboard.getKeyFromScancode('a')
+  test:assertEquals('a', key, 'check key')
```

Comment:
This is dependent on the physical keyboard and system configuration for it, so it shouldn't check for a specific value.